pcweb/templates/docpage.py

```diff
@@ -398,38 +374,28 @@ def docheader2(
     text: str,
     first: bool = True,
     font_size: float = None,
-    coming_soon: bool = False,
-    divider: bool = True,
+    tag="h1",
     **props,
 ) -> rx.Component:
     # Get the basic styles.
     style = {"marginTop": "1em"} if not first else {}
     if font_size:
         style["fontSize"] = font_size
 
-    # Set the text.
-    children = [rx.heading(text, _as="h1", style=style, **props)]
-
-    # Add a badge if the header is coming soon.
-    if coming_soon:
-        children.append(
-            rx.badge(
-                "Coming Soon!",
-                bg=styles.ACCENT_COLOR,
-                color="white",
-            ),
-        )
-
-    # Add a divider if needed.
-    if divider:
-        children.append(rx.divider(margin_y="1em"))
-
     id_ = text[0].to_string(json=False).lower().split().join("-")
 
     # Return the header.
     return rx.box(
-        *children,
-        id=id_,
+        rx.hstack(
+            rx.heading(text, _as=tag, id=id_, style=style, **props),
+            rx.icon(
+                tag="link",
+                style=icon_style,
+                color=styles.ACCENT_COLOR,
+                on_click=lambda: rx.set_clipboard(State.current_page + "#" + id_)
```

Comment:
i think we should make this a real link and set the `href` too, as many people (myself included) right click these guys and "Copy Link"

Reply:
Good call, updated to use a link instead